### PR TITLE
[CELEBORN-1755] Update doc to include S3 as one of storage layers

### PR DESCRIPTION
**Pull request by zhaohehuhu**
### What changes were proposed in this pull request?

as titile

### Why are the changes needed?

The doc fail to mention S3 as one of storage layers

### Does this PR introduce _any_ user-facing change?

No

### CODE REVIEW

**Comment by zhaohehuhu** [comment]
@FMX  plz help review. Thanks.

File: common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala [hunk 1]
```diff
@@ -407,7 +430,6 @@ class CelebornConfSuite extends CelebornFunSuite {
     conf.unset("celeborn.worker.storage.storagePolicy.createFilePolicy")
     val createFilePolicy3 = conf.workerStoragePolicyCreateFilePolicy
     assert(List("MEMORY", "HDD", "SSD", "HDFS", "OSS") == createFilePolicy3.get)
-
```
Comment (turboFei) [inline common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala:410]: nit: unnecessary change

File: common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala [hunk 2]
```diff
@@ -430,7 +452,6 @@ class CelebornConfSuite extends CelebornFunSuite {
     conf.unset("celeborn.worker.storage.storagePolicy.evictPolicy")
     val evictPolicy3 = conf.workerStoragePolicyEvictFilePolicy
     assert(Map("MEMORY" -> List("SSD", "HDD", "HDFS", "OSS")) == evictPolicy3.get)
-
```
Comment (turboFei) [inline common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala:433]: ditto

File: common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala (hunk 1 above)
Comment (zhaohehuhu) [reply common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala]: Reverted.Thanks.

File: common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala (hunk 2 above)
Comment (zhaohehuhu) [reply common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala]: Reverted.Thanks.

**Review by FMX — APPROVED** [review]
LGTM. Will wait for pending discussions.

**Review by turboFei — APPROVED** [review]
